### PR TITLE
Add bitbucket documentation

### PR DESCRIPTION
**Pull request by f-meloni**

### CODE REVIEW

File: docs/usage/bitbucket.html.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- `DANGER_BITBUCKETSERVER_PASSWORD` = The password for the account used to comment.
+- `DANGER_BITBUCKETSERVER_TOKEN` = The personal access token for the account used to comment.
+
+Then in your Dangerfiles you will have a fully fleshed out `danger.bitbucketServer` object to work with. For example:
+
+```kotlin
+val danger = Danger(args)
+
+let isAssignedToMe = danger.bitBucketServer.pullRequest.reviewers.map { it.user.name }.contains("orta")
```
Comment (gianluz) [inline docs/usage/bitbucket.html.md]: Val?

Comment (f-meloni) [reply docs/usage/bitbucket.html.md]: Cool thank you :)

Comment (f-meloni) [reply docs/usage/bitbucket.html.md]: Cool thank you :)

**Comment by f-meloni** [comment]
merge on green

**Pull request merged**